dv/formal/README.md

```diff
@@ -1,23 +1,36 @@
-# End to End Formal Verification against the Sail
+# End-to-End Formal Verification against the Sail
+
 Prerequisites:
 - Nix (`https://zero-to-nix.com/start/install`)
-- JasperGold (available on your PATH)
-  - If JasperGold is not found, `nix develop` will refuse to enter the development shell.
+- JasperGold (available on your PATH):
+  If JasperGold is not found, `nix develop` will refuse to enter the development shell.
 
 Build instructions:
 - Clone this repository
 - `cd dv/formal`
 - `nix develop .#formal`
-- `make` invokes JasperGold in batch-mode, and attempts to prove everything. (for regressions)
+- `make` invokes JasperGold in batch-mode, and attempts to prove everything, which is meant for regressions.
 - `make jg` invokes JasperGold interactively, halting after sourcing the contents of `verify.tcl`.
 
 Building as above invokes the following steps, which can also be done manually during development:
-- `make fusesoc` fetches the necessary RTL using the Fusesoc tool, and makes a local copy inside `build/`. This also creates a filelist (`.scr`) that Jasper knows how to ingest.
-  - This step also patches the Ibex RTL with the changes described [in the section below](#rtl-changes).
-- `git submodule init`
+- `git submodule update --init --recursive`
+- `make fusesoc` fetches the necessary RTL using the FuseSoC tool and makes a local copy inside `build/`.
+  This also creates a filelist (`.scr`) that Jasper knows how to ingest.
+  This step also patches the Ibex RTL with the changes described [in the section below](#rtl-changes).
 - `make psgen` to build the SV for the proofs given in `thm/`
-- `make sv` to build the SV translation of the Sail compiler. Will invoke `buildspec.py`, which can be configured to adjust which instructions are defined. By default all of them are, this is correct but slow.
-- `jg verify.tcl` invokes JasperGold interactively, sourcing the configuration & run script. Requires the above two steps to be executed first.
+- `make sv` to build the SV translation of the Sail compiler.
+  Will invoke `buildspec.py`, which can be configured to adjust which instructions are defined.
+  By default all of them are, this is correct but slow.
+- `jg verify.tcl` invokes JasperGold interactively, sourcing the configuration & run script.
+  Requires the above two steps to be executed first.
+
+After this command, Jasper Gold should start up and should execute the commands in the TCL file.
```

Comment:
Nit: Stray space before Gold. Also, I think Cadence have stopped using that name: isn't it just "Jasper" now?